calibrators/mapping_based_calibrator/README.md

```diff
@@ -0,0 +1,238 @@
+| `frame_path`                      | `nav_msgs::msg::Path`                  | The actual path of the `mapping_pointcloud`.                                                                   |
+| `frame_predicted_path`            | `nav_msgs::msg::Path`                  | The predicted path of the`mapping_pointcloud`.                                                                 |
+| `keyframe_path`                   | `nav_msgs::msg::Path`                  | The keyframe path of the `mapping_pointcloud`.                                                                 |
+| `keyframe_markers`                | `visualization_msgs::msg::MarkerArray` | Markers for keyframes.                                                                                         |
+| `initial_source_aligned_map`      | `sensor_msgs::msg::PointCloud2`        | Initial map from calibration lidars.                                                                           |
+| `calibrated_source_aligned_map`   | `sensor_msgs::msg::PointCloud2`        | Calibrated map from calibration lidars.                                                                        |
+| `target_map`                      | `sensor_msgs::msg::PointCloud2`        | Target map from `mapping lidar`, used for comparing with the `calibrated_source_aligned_map` and `target_map`. |
+| `target_markers`                  | `visualization_msgs::msg::MarkerArray` | Markers for camera calibrator (currently not used)                                                             |
+| `base_lidar_augmented_pointcloud` | `sensor_msgs::msg::PointCloud2`        | Ground pointcloud from augmented pointcloud.                                                                   |
+| `ground_pointcloud`               | `sensor_msgs::msg::PointCloud2`        | Ground pointcloud from calibrated pointcloud.                                                                  |
+
+### Services
+
+| Name                    | Type                                                  | Description                                                                              |
+| ----------------------- | ----------------------------------------------------- | ---------------------------------------------------------------------------------------- |
+| `extrinsic_calibration` | `tier4_calibration_msgs::` `srv::ExtrinsicCalibrator` | Generic calibration service. The call is blocked until the calibration process finishes. |
+| `stop_mapping`          | `std_srvs::srv::Empty`                                | Stops building the map; calibration will begin afterward.                                |
+| `load_database`         | `std_srvs::srv::Empty`                                | Loads lidar and camera calibration frames from the database. (currently not used)        |
```

Comment:
It is actually used, but mostly for developers

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/1a6dc00efc713156605475161b5aeb87ab47366a